MapboxNavigation/StyleManager.swift

```diff
@@ -97,7 +97,24 @@ open class StyleManager: NSObject {
     }
     
     func applyStyle() {
-        guard let location = delegate?.locationFor(styleManager: self) else { return }
+        guard let location = delegate?.locationFor(styleManager: self) else {
+            // We can't calculate sunset or sunrise w/o a location so just apply all styles
+            for style in styles {
+                style.apply()
+                delegate?.styleManager?(self, didApply: style)
```

Comment:
In the case where we cannot find a style, we're going to rapid fire go through all supplied styles and apply them? What if we just applied the first?

Reply:
I assume styles.count would mostly be 1 but I could imagine e.g. `[TopBarDayStyle(), BottomBarDayStyle(), FloatingButtonsDaysStyle()]` if one class would become unwieldy. 🤔